docs/ReleaseProcess.md

```diff
@@ -0,0 +1,89 @@
+# Release Process: openlaw-core
+
+Note this documentation is for core project maintainers and all steps may not apply to open source contributors. If in doubt, just ask! We're friendly folks. :rainbow:
+
+## Submitting Changes :sparkles:
+
+New code changes should always be introduced via a Pull Request branch:
+
+1. Pull latest `master` branch then create new branch off from `master` and make the changes there.
+
+2. If added any new sbt tests, make sure they are all passing.
+
+3. Submit PR for merging that new branch into `master`. GitHub Actions will automatically attempt a build and run tests, which you can see here: https://github.com/openlawteam/openlaw-core/actions.
+
+4. If the build and all checks are passing, the PR can be merged to master.
+
+## Releasing to BinTray :rocket:
+
+The `master` branch represents the current stable set of the code, and should be considered "release-able" at any point in time.
+
+New _releases_ are cut from the `master` branch, and managed via GitHub Releases with [Semantic Versioning](https://semver.org/spec/v2.0.0.html) formatted tags. <!-- As a developer, this process is largely automated for you. -->
```

Comment:
should we add - "This process will be largely automated for you"?

Reply:
or just remove the above?

Reply:
Ah, I commented it out so it wouldn't show up when viewing the rendered markdown README, but could be pre-written and easily uncommented out once we "go live". I think I have a few things like that in there, hehe.